function_requirements/common-requirements.in

```diff
@@ -1,7 +1,7 @@
 boto3
 aws-lambda-logging
 requests
-PyJWT
+PyJWT>=2.0.1
```

Comment:
Behavior changed in this package between 1.X and 2.X. jwt.encode() now returns str not bytes